build/azure-pipeline.pre-release.yml

```
@@ -15,12 +15,32 @@ resources:
     - repository: templates
       type: github
       name: microsoft/vscode-engineering
-      ref: main
+      ref: tyler/translations
       endpoint: Monaco
 
 extends:
   template: azure-pipelines/extension/pre-release.yml@templates
   parameters:
+    locPushSteps:
+      - script: echo '$(SetTranslationsPath.EnglishStrings)'
+      - script: npm ci
+        displayName: Install NPM dependencies
+      - script: npm run translations-export
+        displayName: Run translations-export
+      - pwsh: |
+          $Destination = "$(SetTranslationsPath.EnglishStrings)"
+          Move-Item "$(Pipeline.Workspace)/ms-python.python/*" $Destination -Force
+        displayName: Move vscode-python-translations-export to translations path
```

```suggestion
      - pwsh: Move-Item "$(Pipeline.Workspace)/ms-python.python/*" "$(SetTranslationsPath.EnglishStrings)" -Force
        displayName: Move vscode-python-translations-export to translations path
```

slightly more concise.